deploy/docker-compose-jdbc.yml

```diff
@@ -1,38 +0,0 @@
-# Additional services needed to test/demo the Debezium JDBC connector.
-
-services:
```

Comment:
these deleted docker compose files are referenced from demo scripts, e.g. https://github.com/feldera/feldera/blob/7b87c387442b1ad1cedb38cfe197816c172daa8a/demo/project_demo08-DebeziumJDBC/run.py#L4

Please fix up those references too (but no need to actually try those command lines)

Reply:
Fixed up references as far as I could find